index.html

```diff
@@ -18,7 +18,6 @@
 </h3>
 <p>
   Tedious is pure Javascript, as are its dependencies.
-  (Well, the source is <a href="http://jashkenas.github.com/coffee-script/">CoffeeScript</a>.)
   So it should run without change on any platform where Node is supported.
```

Comment:
Only suggestion I'd have is mention the minimum Node version required (since Tedious is using ES2015+)